docs/openstack/advanced-openstack-topics/data-transfer/data-transfer-from-to-vm.md

```diff
@@ -0,0 +1,568 @@
+
+- WinSCP installed, see [Download and Install the latest version of the WinSCP](https://winscp.net/eng/docs/guide_install)
+for more information.
+
+- Go to WinSCP menu and open "View > Preferences".
+
+- When the "Preferences" dialog window appears, select "Transfer" in the options
+on the left pane.
+
+- Click on "Edit" button.
+
+- Then, on shown popup dialog box review the "Common options" group, uncheck the
+"Preserve timestamp" option as shown below:
+
+![Disable Preserve TimeStamp](images/winscp-preferences-perserve-timestamp-disable.png)
+
+#### Configuring WinSCP
+
+- Click on "New Tab" tab button as shown below:
+
+![Login](images/winscp-new-tab.png)
+
+- Select either **"SFTP"** or **"SCP"** from the "File protocol" dropdown options
+as shown below:
+
+![Choose SFTP or SCP File Protocol](images/choose_SFTP_or_SCP_protocol.png)
+
+- Provide the following required information:
+
+    **"File protocol"**: Choose either "**"SFTP"** or **"SCP"**"
+
+    **"Host name"**: "`<Your Floating IP of VM>`"
+
+    **"Port number"**: "22"
+
+    **"User name"**: "`<Default User name based on OS>`"
+
+    !!! info "Default User name based on OS"
+        - **all Ubuntu images**: ubuntu
+        - **all CentOS images**: centos
+        - **all Rocky Linux images**: centos
+        - **all Fedora images**: fedora
+        - **all Debian images**: debian
+        - **all RHEL images**: cloud-user
+
+    **"Password"**: "`<Leave blank as you using SSH key>`"
+
+- Change Authentication Options
+
+Before saving, click the "Advanced" button.
+In the Advanced Site Settings, under SSH >> Authentication settings check
+“Allow agent forwarding” and select Private key file with `.ppk` extension from
+the file picker.
+
+![Advanced Site Settings for SSH Authentication](images/winscp-ssh-auth.png)
+
+!!! tip "Helpful Tip"
+    You can save your above configured site with some preferred name by
+    clicking the "Save" button and then giving a proper name to your site.
+    So that next time you don't need to again manually enter all your configuration.
+    ![Save Site WinSCP](images/winscp-save-site.png)
+
+#### Using WinSCP
+
+You can follow above step to manually add a new site next time you open WinSCP
+or, you can connect to your previously saved site (as listed on popup dialog
+will show your all saved site name list) that will show up by just clicking on
+the site name.
+
+Then click "Login" button to connect to your NERC project's VM as shown below:
+
+![Login](images/winscp-site-login.png)
+
+![Successful connection](images/winscp-site-successfully-connected.png)
+
+You should now be connected to the VM's remote directories/files. You can drag
+and drop your files to/from file windows to begin transfer. When done, click the
+X icon up top right to disconnect.
+
+### ii. Cyberduck
+
+[Cyberduck](https://docs.cyberduck.io/cyberduck/) is a libre server and cloud
+storage browser for Mac and Windows. With an easy-to-use interface, connect to
```

Comment:
Not sure about the "connect to servers" part here - consider rewording or removing

Reply:
Its user-friendly interface enables seamless
connections to servers, enterprise file sharing, and various cloud storage platforms.